README.md

```diff
@@ -1376,8 +1376,11 @@ ANSWERS.addComponent('Map', {
   onLoaded: function () {},
   // Optional, configuration for the map's behavior when a query returns no results
   noResults: {
-    // Optional, whether to display all results in the vertical when no results are found. Defaults to false, in which case only the no results card will be shown.
-    displayAllResults: true
+    // Optional, whether the map should be visible when displaying no results.
```

Comment:
I think this is pretty misleading given that it actually still determines whether to display all results. The above description would fit more for showEmptyMap.

Reply:
sorry one sec I misread something

Reply:
updated!